docker/production/production.Dockerfile

```diff
@@ -0,0 +1,107 @@
+FROM nvidia/cuda:12.6.2-devel-ubuntu22.04 AS devel
```

Comment:
Maybe put a comment at the top not to edit this file since it's automatically generated by hpccm